libraries/botbuilder-dialogs/botbuilder/dialogs/dialog_extensions.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from botbuilder.core import BotAdapter, StatePropertyAccessor, TurnContext
+from botbuilder.dialogs import (
+    Dialog,
+    DialogEvents,
+    DialogSet,
+    DialogTurnStatus,
+)
+from botbuilder.schema import Activity, ActivityTypes
+from botframework.connector.auth import ClaimsIdentity, SkillValidation
+
+
+class DialogExtensions:
+    @staticmethod
+    async def run_dialog(
+        dialog: Dialog, turn_context: TurnContext, accessor: StatePropertyAccessor
+    ):
+        dialog_set = DialogSet(accessor)
```

Comment:
Does dialogs in python support the TelemetryClient property? if yes, we need to set it.

In C# this line looks as:

```
var dialogSet = new DialogSet(accessor) { TelemetryClient = dialog.TelemetryClient };
```

Reply:
It does not appear DialogSet supports telemetry client yet.  At least it's commented out.  I'll open a parity issue